rct229/rulesets/ashrae9012019/section19/section19rule18.py

```diff
@@ -172,9 +174,7 @@ def get_calc_vals(self, context, data=None):
             )
             supply_flow_b = fan_sys_info_b["supply_fans_airflow"].to(ureg.cfm)
 
-            more_than_one_supply_fan_b = (
-                fan_sys_info_b["supply_fans_qty"] != 1
-            )  # the reason why > 1 isn't used is to prevent an error when no supply fan exists
+            exactly_one_supply_fan = fan_sys_info_b["supply_fans_qty"] == 1
```

Comment:
Does no supply fan a concern in this rule? I saw it is captured by manual check so we should be fine with this logic.

Reply:
This is related to my initial comment on the PR. I noticed in our test case for this scenario that the logic goes to `table_G3_9_1_lookup` and results in  `RCTFailureException: The 'shaft_input_power' must be greater than 0.0 hp.`

Reply:
I am not sure if we want to handle this scenario more explicitly

Reply:
I see, the failure comes from missing supply flows - this is a typical data errors we should capture in the RCT. I do think it makes sense to leave it as is so the data look up function captures the issue.